packages/mux-player/src/time-display.ts

```diff
@@ -0,0 +1,69 @@
+const styles = `
+  :host {
+    cursor: pointer;
+  }
+`;
+
+const template = document.createElement("template");
+template.innerHTML = `
+  <style>
+    ${styles}
+  </style>
+  <media-time-display show-duration></media-time-display>
```

Comment:
non-blocking: proposed "extending" `media-time-display` instead of using it as a descendant in the template, but that's not a high priority imo. fwiw I suspect if we wanted this functionality migrated over to Media Chrome, the code would more closely resemble "extending".

Reply:
yeah was thinking we shouldn't over architect it if this is expected to make its way into the built in functionality of `media-time-display`

Reply:
Sure, though, this does beg the question, can/should you extend media-chrome-button, and are the chrome buttons purely for icons or can they be used with text too? I.e., if someone wanted to make a custom button, what's their workflow like? Though, definitely a media-chrome question. I can raise a discussion there.